bundle/Resources/init_ini/siteaccess/legacy_admin/site.ini.append.php

```diff
@@ -0,0 +1,33 @@
+<?php /* #?ini charset="utf-8"?
+
+[SiteSettings]
+DefaultPage=content/dashboard
+LoginPage=custom
+
+[SiteAccessSettings]
+RequireUserLogin=true
+RelatedSiteAccessList[]
+RelatedSiteAccessList[]=site
+#RelatedSiteAccessList[]=admin
+RelatedSiteAccessList[]=legacy_admin
+ShowHiddenNodes=true
+
+[DesignSettings]
+SiteDesign=admin2
+AdditionalSiteDesignList[]
+AdditionalSiteDesignList[]=admin
+AdditionalSiteDesignList[]=standard
+AdditionalSiteDesignList[]=base
+
+[RegionalSettings]
+Locale=eng-GB
```

Comment:
You could in theory make this a Twig template and inject the correct values taken from Symfony container. Same goes for the list of siteaccesses.

Similar to this: https://github.com/netgen/NetgenAdminUIBundle/blob/master/bundle/Installer/_templates/legacy_siteaccess/site.ini.append.php#L23

Reply:
Ideally yes.

Is that something you1'd be able to contribute here? _(the code involved in the "rendering" and saving of the results of the ini templates)_

Reply:
I could, but next only next week :+1:

Reply:
@emodric can we do this as a followup? Given this is meant to be used for testing use cases straight after clean install, and atm clean install still is limited to eng-GB, this is imho ok for a start.

Reply:
Sure, no problem.

Reply:
@emodric I'm unable to figure out the proc_open issue on PHP 5.6 omn Travis, could you review this regardless of 5.6 failure?